code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -346,6 +346,7 @@ GLOBAL_LIST_INIT(rarity_loot, list(//rare: really good items
 		/obj/item/book/granter/crafting_recipe/maint_gun/pipegun_prime = 1,
 		/obj/item/book/granter/crafting_recipe/trash_cannon = 1,
 		/obj/item/book/granter/crafting_recipe/maint_gun/laser_musket_prime = 1,
+		/obj/item/book/granter/crafting_recipe/maint_gun/smoothbore_disabler_prime = 1,
```

Comment:
excluded for a reason when i added these to maints.

Reply:
Why is that?

Reply:
because its very powerful and you can get the book through black market instead(rarely)

Reply:
I've never seen them be an issue, and there's nothing wrong with there being more than one way to get it. Cargo's the only people who can order the black market transceiver thing I think, so giving others a method to get one is nice.